src/controllers/identity/policy/index.ts

```diff
@@ -30,9 +30,31 @@ const getPolicy = async (params) => {
 
 const listPolicies = async (params) => {
     const identityV1 = await grpcClient.get('identity', 'v1');
-    const response = await identityV1.Policy.list(params);
+    const identityResponse = await identityV1.Policy.list(params);
+    
+    /*1. Collect all repository_id
+    2. Collect repository.Policy with the set of repository_id
+    3. Merge repository.Policy into identity.Policy*/
+    const repositoryV1 = await grpcClient.get('repository', 'v1');
+    const repositories = await repositoryV1.Repository.list(params);
+    const repositoryIDList = repositories.results.map((d) => (d.repository_id));
+    const repositoryPromise = repositoryIDList.map((d) => { return repositoryV1.Policy.list({ ...params, repository_id: d }); });
+    const repositoryResponse = await Promise.allSettled(repositoryPromise);
 
-    return response;
+    const response: Array<any> = [];
+    identityResponse.results.map(d => response.push(d));
+    repositoryResponse.map(d => {
```

Comment:
```suggestion
    repositoryResponse.forEach(d => {
```

리턴 결과 배열을 받아 사용하지 않는다면, forEach를 사용하는 것이 가독성을 높이는 좋은 방법이 될 것 같습니다.